src/getOperands.js

```diff
@@ -26,7 +26,13 @@
  * @return {string} - operands
  */
 function getOperands(str) {
-  // write code here
+  const regEx = /-?\d\.?\d*\s(\+|-|\*|\/)\s-?\d\.?\d*/g;
+  const regExToSplit = /\s\D\s/g;
+  const matchArray = str.match(regEx).join('').split(regExToSplit);
```

Comment:
массив чего-то - множественное число переменной
```suggestion
  const matches = str.match(regEx).join('').split(regExToSplit);
```